__tests__/app.test.js

```diff
@@ -76,32 +76,45 @@ describe("4. GET /api/articles", () => {
   });
 });
 
-describe("5. GET /api/articles/:article_id", () => {
-  it("status:200, should responds with an article object", () => {
+describe("5 & 11. GET /api/articles/:article_id with comment count", () => {
```

Comment:
Ideally you'd write a new test, instead of amending the old one - as its a new feature